Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -93,3 +93,18 @@
     - type: AddAccentClothing
       accent: ReplacementAccent
       replacement: mumble
+
+- type: entity
+  parent: ClothingMaskBase
+  id: ClothingMaskPlague
+  name: plague doctor mask
+  description: A sight of bad omen
```

Comment:
Should it be sign of a bad omen?

Reply:
Omen literally means sign so that's also a bit redundant